Off-Top-App/off-top-flutter
# Ot-94 | FE - Upon Login, hit a GET request that returns user information with that user's entered email.

What I did: Created a GET request that returns the users information upon logging in.

How to test: 

- Have a dummy user in the database.
Refer to:
https://docs.google.com/document/d/1yDnXPHTTi3lG7JpxipsTzAXDrPxpBJcawUWIGjWV8sM/edit

- Have zookeeper and kafka running.

- Open up emulator and do a flutter run.

- Click sign in and it should be able to print out the userId in the terminal and then it will log you in to the main page.
